script.js

```diff
@@ -0,0 +1,16 @@
+// Select Form
+const form = document.getElementById('resumeInputForm')
+
+// const nameInput = document.getElementById('formGroupNameInput')
+
+// let name = nameInput.value
+// console.log(nameInput)
+
+// Methods
+
+const handleFormSubmit = event => {
+  event.preventDefault()
+  const data = {}
+  const dataContainer = document.getElementsByClassName('dataContainer')
+  dataContainer.textContent = JSON.stringify(data)
```

Comment:
Like the simplicity of this code to get the proper format. Consider how you might handle multiple inputs with arrays if you use this approach.